CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - Bugfix: Fixed a bug that caused all badge highlights to use the same color. (#3132, #3134)
 - Bugfix: Allow starting Streamlink from Chatterino when running as a Flatpak. (#3178)
 - Bugfix: Fixed own IRC messages not having metadata and a link to a usercard. (#3203)
+- Bugfix: Fixed /streamlink command not stripping leading @'s or #'s (#3215)
```

Comment:
minor not bugfix, same as original PR positioning

Reply:
We should also have /streamlink enclosed in backticks.